gulpfile.js

```diff
@@ -65,6 +66,15 @@
 			source.push("!" + path.join(__dirname, componentsToBuild[key].entry));
 		}
 
+		//Dont copy files built by angular
+		try {
+			var angularComponents = require('./build/angular-components.json');
+			var arrayLength = angularComponents.length;
+			for (var i=0; i < arrayLength; i++) {
+				source.push(path.join('!' + __dirname, angularComponents[i].source, '**'));
+			}
+		} catch (ex) {}
```

Comment:
Log error

Reply:
the only expect error was the require failing as file didn't exist. Gone a different way now.